config.py

```diff
@@ -1,4 +1,14 @@
 import os
+import sys
 
 DEBUG = True
 BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+SECRET_KEY = os.environ.get('MOZ_SECRET_KEY', 'SUPER_SECRET_KEY')
+
+DB_NAME = 'sql11223417'
```

Comment:
Whould be better if all db properties will be stored in environment, for simplicity we can provide to some of them default values  os.environ.get('MOZ_DB_NAME', 'sql11223417')